.github/workflows/ui-rm_prime_alpha_rc.yaml

```diff
@@ -0,0 +1,59 @@
+# This workflow calls the master E2E workflow with custom variables
+name: UI-RM_Prime_Alpha_RC
+run-name: ${{ github.event_name == 'workflow_dispatch' && format('`{0}` on `{1}` - `{2}` destroy={3}', inputs.rancher_version, inputs.upstream_cluster_version, inputs.grep_test_by_tag, inputs.destroy_runner) || github.workflow }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      qase_run_id:
+        description: Qase run ID where the results will be reported. (Please update with custom run ID.)
+        default: none
+        type: string
+      destroy_runner:
+        description: Destroy the auto-generated self-hosted runner
+        default: true
+        type: boolean
+      rancher_version:
+        description: Rancher version channel/version/head_version latest/2.9.x[-rc1], prime/2.9.x, prime/devel/2.9, alpha/2.9.0-alpha1
```

Comment:
Non blocker, but perhaps to consider for further iterations 
`alpha/2.9.0-alpha1` to `alpha/2.x.x-alphax`

Reply:
Updated it.